libs/shared/src/types/overlay/overlay.ts

```diff
@@ -14,6 +14,7 @@ export interface ChatOverlayOptions {
   newConversationsFolderId?: string;
 
   enabledFeatures?: Feature[] | string;
+  enabledFeaturesData?: Partial<Record<Feature | string, FeatureData>>;
```

Comment:
Why should we allow string here? Only Feature keys I suppose
```suggestion
  enabledFeaturesData?: Partial<Record<Feature, FeatureData>>;
```

Reply:
As you can see features above is Feature or string, so I did it for consistency

Reply:
we shouldn't allow string as a key, only `Feature`

Reply:
I don't agree - we already supported this and it will be breaking change